build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.7.10"
+    application
+}
+
+group = "org.stellar"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+    maven("https://jitpack.io")
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+    implementation("com.github.stellar:java-stellar-sdk:0.34.2")
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.jvmTarget = "1.8"
```

Comment:
1.8 is a little low. It should be safe to use 1.11 which is more modern and is one of the most popular version for backward compatibility.